js/leaflet-sidebar.js

```diff
@@ -72,9 +72,12 @@ L.Control.Sidebar = L.Control.extend(/** @lends L.Control.Sidebar.prototype */ {
 
         for (i = this._tabitems.length - 1; i >= 0; i--) {
             child = this._tabitems[i];
-            L.DomEvent
-                .on(child.querySelector('a'), 'click', L.DomEvent.preventDefault )
-                .on(child.querySelector('a'), 'click', this._onClick, child);
+            if (child.querySelector('a')) {
```

Comment:
can you extract `child.querySelector('a')` into a local variable so that we don't execute the query multiple times?